README.md

```diff
@@ -35,7 +35,7 @@
   - [Debugging Puppet manifests](#debugging-puppet-manifests)
   - [Docker Language Server Support](#docker-language-server-support)
 - [Installing the Extension](#installing-the-extension)
-- [Configuration](#configuration)
+- [Configuration](#**configuration**)
```

Comment:
Is this supposed to be bolded?

Reply:
Today I Remembered that `ctrl+b` in markdown files adds bold to text and overrides the hide sidebar command in vscode.

Reply:
fixed